src/server/config.js

```diff
@@ -56,6 +56,9 @@ export default function (baseConfig, configDir) {
     module: {
       ...config.module,
       // We need to use our and custom loaders.
+      preLoaders: [
```

Comment:
Hmm. Did you try that, it's add loaders but in the line 62 it'll get override again with the our own loaders?
If that's not working, I'll take this in.